### PR TITLE
Feature/change poltergeist to selenium

### PR DESCRIPTION
**Pull request by renanbona**
- Remove Poltergeist
- Add Selenium
- Refactor Tests

### CODE REVIEW

**Comment by kostadriano** [comment]
@renanbona there are some errors in tests, can you check [this](https://travis-ci.org/Codeminer42/cm42-central/builds/489749542?utm_source=github_status&utm_medium=notification)?

File: config/environments/test.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@
   # Randomize the order test cases are executed.
   config.active_support.test_order = :random
 
-  config.middleware.use NoAnimations
+  # config.middleware.use NoAnimations
```
Comment (talyssonoc) [inline config/environments/test.rb]: If it's not needed anymore we can probably remove this line.

File: spec/features/keycut_spec.rb [hunk 2]
```diff
@@ -39,7 +39,6 @@
 
     it 'adds story (a)', js: true do
       wait_page_load
-      click_on 'Chilly Bin'
```
Comment (talyssonoc) [inline spec/features/keycut_spec.rb]: Why was this needed?

File: spec/support/wait_for_ajax.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module WaitForAjax
+  def wait_for_ajax
```
Comment (talyssonoc) [inline spec/support/wait_for_ajax.rb]: We gotta avoid using this. `wait_for_ajax` relies on the fact that we are using `jQuery` which isn't the case for the v2 board.

File: spec/features/notes_spec.rb [hunk 4]
```diff
@@ -23,12 +23,12 @@
...
 
       within('#in_progress .story') do
-        find('.story-title').trigger('click')
+        find('.story-title').click
         fill_in 'note', with: 'Adding a new note'
         click_on 'Add note'
       end
 
-      sleep 0.5
+      wait_for_ajax
```
Comment (talyssonoc) [inline spec/features/notes_spec.rb]: We can have something similar to the `wait_spinner` here for the edit state that disables the fields while saving. We have it for v1 as well, right @kostadriano?

File: spec/features/keycut_spec.rb (hunk 2 above)
Comment (renanbona) [reply spec/features/keycut_spec.rb]: That was necessary because, when we click on Chilly Bin the keycut stops working. To be honest I don't know why this test was passing before (when we used poltergeist). The correct approach is to press the keycut when we don't have anything selected.

File: spec/support/wait_for_ajax.rb (hunk 3 above)
Comment (renanbona) [reply spec/support/wait_for_ajax.rb]: Ok, I'll remove this and try the `wait_spinner` and see if it works.
